corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -107,6 +107,10 @@ hqDefine("app_manager/js/details/screen", function () {
             const caseTileTemplate = self.caseTileTemplate() && self.caseTileTemplate() !== "custom";
             return caseTileTemplate && featureFlag;
         });
+        self.showCaseListOptimizations = (
+            self.columnKey === 'short' &&
```

Comment:
"short" is to refer to "case list".
"long" is used for "case detail".

This change is relevant only for case list.